src/Components/Contacts.js

```diff
@@ -0,0 +1,15 @@
+import ContactUserCard from "/Users/EricCave_1/Documents/pursuit_labs/8-0-react-introduction-lab/src/Components/ContactUserCard.js";
```

Comment:
Here it would be import ContactUserCard from "./ContactUserCard.js"
We use the "./" to show that the file we want is in the same file path or folder
If we did "../" it would be one folder out
if we did "../../" it would be to folders out of our current file